fractal_tasks_core/tasks/create_ome_zarr.py

```diff
@@ -428,14 +428,14 @@ def create_ome_zarr(
                 "FOV_ROI_table",
                 FOV_ROIs_table,
                 overwrite=overwrite,
-                logger=logger,
+                table_attrs={"type": "roi_table"},
```

Comment:
If the type is the only kind-of required attrs: It's a bit hidden to have it in the optional attrs

(but very useful to pass it through there when just taking over the attrs of an existing table, so not sure I'd argue against this)